WordPress/gutenberg
# Flatten blocks directory

Previously: #716, #736
Related: #739

This pull request expands upon the restructuring outlined in #716, eliminating a blocks-specific `components` directory in favor of flattening the sole component into the top-level `blocks` directory.

__Testing instructions:__

Ensure the build completes, tests pass, and editor initializes and appears without regressions. Specifically observe that editable text regions retain appearance and behavior.
